visualization/callbacks/__init__.py

```diff
@@ -0,0 +1,49 @@
+import os
+from pathlib import Path
+
+import dash_bootstrap_components as dbc
+import hydra
+from dash import Dash
+from flask import Flask
+from omegaconf import OmegaConf
+from settings.config import Config
+
+from nemo_skills.utils import unroll_files
+
+config_path = os.path.join(os.path.abspath(Path(__file__).parent.parent), "settings")
+
+config = {}
+
+
+@hydra.main(version_base=None, config_path=config_path, config_name="config")
+def set_config(cfg: Config) -> None:
+    global config
+    config['prompt_explorer'] = OmegaConf.to_container(cfg)
```

Comment:
Let's not call it "prompt_explorer", but "data_explorer", since it's not just for prompt, right?